production-deployment-using-tiup.md

```diff
@@ -83,7 +83,9 @@ Perform the following steps in this section to deploy a TiDB cluster offline usi
 
 #### Step 1: Prepare the TiUP offline component package
 
-To prepare the TiUP offline component package, manually pack an offline component package using `tiup mirror clone`.
+Option 1: Select the corresponding version of the package of the TiDB server's offline mirrors (including the TiUP offline component package) on the [Download TiDB page](https://pingcap.com/download/).
```

Comment:
Although the former version does not have this part, I added it temporarily because the updated part that needed translation mentioned it. If we don't need this one, should we also revise the translation "If the package of the TiDB server's offline mirrors from the PingCAP website does not meet your specific needs," in the below part?

Reply:
This part is not needed in the English version. You cannot "Select the corresponding version of the package" in https://pingcap.com/download/. The English version only provides the `tiup mirror clone` download method. Please translate the docs-cn PR **AS IT IS**.